check_nic

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+STATE_OK=0
```

Comment:
```suggestion
PROGPATH='/usr/lib/nagios/plugins'

. $PROGPATH/utils.sh
```

Reply:
IMHO state codes are less likely to change than nagios plugins location that may be different on some distros.